.circleci/config.yml

```diff
@@ -38,177 +38,232 @@ defaults: &defaults
-                command: composer install
-
-            - run:
-                name: run functional tests with Behat
-                command: ./.ci/scripts/03-test
-
-            - run:
-                name: post-test actions
-                command: ./.ci/scripts/04-post-test
-
-            - run:
-                name: handle merge to master (if needed)
-                command: ./.ci/scripts/05-merge-master
-
-            - run:
-                name: remove transient test fixtures
-                command: ./.ci/scripts/09-cleanup-fixtures
-
-    composer_lock_updater:
-        <<: *defaults
-        docker:
-            - image: quay.io/pantheon-public/build-tools-ci:5.x
-        steps:
-            - checkout
-
-            - run:
-                name: setup-environment-vars
-                command: /build-tools-ci/scripts/set-environment
-
-            - run:
-                name: run composer lock updater
-                command: set -e && clu https://${GITHUB_TOKEN}:x-oauth-basic@github.com/${CI_PROJECT_USERNAME}/${CI_PROJECT_REPONAME}.git | sed -e "s/${GITHUB_TOKEN}/[REDACTED]/g"
-
-    visual_regression_test:
-        <<: *defaults
-        docker:
-            - image: backstopjs/backstopjs:3.9.5
-        steps:
-            - checkout
-
-            - attach_workspace:
-                at: /tmp/workspace
-
-            - run: cp /tmp/workspace/bash_env.txt $BASH_ENV
-            - run: echo "export CI_BUILD_URL='${CIRCLE_BUILD_URL}'" >> $BASH_ENV
-            - run: echo "export CI_NODE_INDEX='${CIRCLE_NODE_INDEX}'" >> $BASH_ENV
-            - run: source $BASH_ENV
-
-            - run:
-                name: visual regression test
-                command: ./.ci/visual-regression-test.sh
-
-            - store_artifacts:
-                path: /tmp/artifacts
-                destination: artifacts
+  configure_env_vars:
+    <<: *defaults
+
+    steps:
+      - checkout
+
+      - run:
+          # Set TERMINUS_ENV and related environment variables.
+          # https://github.com/pantheon-systems/docker-build-tools-ci/blob/6.x/scripts/set-environment
+          name: setup-environment-vars
+          command: /build-tools-ci/scripts/set-environment
+
+      - run:
+          name: clone-bash-env
+          command: cp $BASH_ENV bash_env.txt
+
+      # https://circleci.com/docs/2.0/workflows/#using-workspaces-to-share-data-among-jobs
+      - persist_to_workspace:
+          root: .
+          paths:
+            - bash_env.txt
+
+  static_tests:
+    <<: *defaults
+
+    steps:
+      - checkout
+
+      - restore_cache:
+          keys:
+            - composer-cache-{{ checksum "composer.lock" }}
+            - composer-cache-
+
+      - run:
+          # Set TERMINUS_ENV and related environment variables.
+          # https://github.com/pantheon-systems/docker-build-tools-ci/blob/6.x/scripts/set-environment
+          name: setup-environment-vars
+          command: /build-tools-ci/scripts/set-environment
```

Comment:
Since we're bothering to save $BASH_ENV, maybe we should make this step depend on configure_env_vars so that we can be consistent about including it here?

Reply:
> Since we're bothering to save $BASH_ENV, maybe we should make this step depend on configure_env_vars so that we can be consistent about including it here?

Considering `/build-tools-ci/scripts/set-environment` takes less than ` second to run I don't think there is anything to be gained here.